exaudfclient/base/javacontainer/BUILD

```diff
@@ -3,38 +3,48 @@ package(default_visibility = ["//visibility:public"])
 genrule(
     name = "exascript_java_tmp_cc",
     cmd = """
-        mkdir -p $(location java_src)/com/exasol/swig        
+        mkdir -p java_src/com/exasol/swig
         mkdir -p build_exascript_java_tmp_cc/exaudflib
         cp "$(location //exaudflib:swig/script_data_transfer_objects_wrapper.h)" "$(location //exaudflib:exascript.i)" build_exascript_java_tmp_cc/exaudflib
-        cd build_exascript_java_tmp_cc     
-        swig -v -O -DEXTERNAL_PROCESS -Wall -c++ -java -addextern -module exascript_java -package com.exasol.swig -outdir "../$(location java_src)/com/exasol/swig" -o "../$(location exascript_java_tmp.cc)" exaudflib/exascript.i
-        """,
-    outs = ["exascript_java_tmp.cc","java_src"],
+        cd build_exascript_java_tmp_cc
+        swig -v -O -DEXTERNAL_PROCESS -Wall -c++ -java -addextern -module exascript_java -package com.exasol.swig -outdir "../java_src/com/exasol/swig" -o "../exascript_java_tmp.cc" exaudflib/exascript.i
+        cd ..
+        tar cf java_src.tar ./java_src
+        cp java_src.tar $(location java_src.tar)
+        cp exascript_java_tmp.cc $(location exascript_java_tmp.cc)
+    """,
+    outs = ["java_src.tar",
+            "exascript_java_tmp.cc"
+    ],
     srcs = ["//exaudflib:exascript.i","//exaudflib:swig/script_data_transfer_objects_wrapper.h"]
 )
 
 genrule(
     name = "exascript_java_tmp_h",
     cmd = """
         mkdir build_exascript_java_tmp_h
+        cp $(location :java_src.tar) .
+        tar xf java_src.tar
         cp "$(location //exaudflib:swig/script_data_transfer_objects_wrapper.h)" "$(location //exaudflib:exascript.i)" build_exascript_java_tmp_h
-        cp -r "$(location exascript_java_tmp.cc)"  "$(location java_src)" build_exascript_java_tmp_h
+        cp -r "$(location exascript_java_tmp.cc)" java_src build_exascript_java_tmp_h
         cd build_exascript_java_tmp_h
         swig -v -DEXTERNAL_PROCESS -c++ -java -external-runtime "../$(location exascript_java_tmp.h)"
         """,
     outs = ["exascript_java_tmp.h"],
-    srcs = ["//exaudflib:exascript.i","//exaudflib:swig/script_data_transfer_objects_wrapper.h", ":exascript_java_tmp.cc", ":java_src"]
+    srcs = ["//exaudflib:exascript.i","//exaudflib:swig/script_data_transfer_objects_wrapper.h", ":exascript_java_tmp.cc", ":java_src.tar"]
 )
 
 genrule(
     name = "exascript_java_int",
     cmd = """
-        cp -r $(locations :exascript_java_tmp_cc) .
+        cp $(locations :java_src.tar) .
+        tar xf java_src.tar
+        ls -la java_src
         FILES=`ls java_src/com/exasol/swig/*.java`
         python3 $(location //:build_integrated.py) "$(location exascript_java_int.h)" $$FILES
     """,
     outs = ["exascript_java_int.h"],
-    srcs = [":exascript_java_tmp_cc"],
+    srcs = [":java_src.tar"],
     tools = ["//:build_integrated.py"]
 )
```

Comment:
remove rule, integrated java source is not used in javacontainer.cc, only the jar is loaded

Reply:
Removed rule `exascript_java_int` completely. Seems it works.... :pray: :pray: :pray: